crates/bevy_text/src/text2d.rs

```diff
@@ -22,17 +22,10 @@ use bevy_utils::HashSet;
 use bevy_window::{WindowId, WindowScaleFactorChanged, Windows};
 
 use crate::{
-    Font, FontAtlasSet, FontAtlasWarning, HorizontalAlign, Text, TextError, TextLayoutInfo,
-    TextPipeline, TextSettings, VerticalAlign, YAxisOrientation,
+    Font, FontAtlasSet, FontAtlasWarning, Text, TextError, TextLayoutInfo, TextPipeline,
+    TextSettings, YAxisOrientation,
 };
 
-/// The calculated size of text drawn in 2D scene.
-#[derive(Component, Default, Copy, Clone, Debug, Reflect)]
-#[reflect(Component)]
-pub struct Text2dSize {
```

Comment:
I see we are losing the "calculated size of the text" as something the user can read. 

Is this desired? 
Does the `Text2DBounds` provide the same data so that it is not necessary?

Reply:
TextLayoutInfo stores the text size too. Text2dSize should have been removed when TextLayoutInfo was added, but it was missed I guess because it was also being used as a marker component for the Text2d query.